### ISSUE TITLE
ensure that nginx directories are created

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/mantl#1459]
- Git commit hash or branch: 61e683f1c155567c92a0fbf51d62c957a856a51f
- Cloud Environment:  Openstack
- Terraform version (`terraform version`): 0.6.16
- Ansible version 1.9.4

Ensure that for mesos and other nginx ui tasks that the /etc/nginx directory exists
```
/etc/nginx
/etc/nginx/templates
```

**Comment by vallard** [comment]
yeah, just ran into this too.  Thanks for working on it @siddharthist

**Comment by vallard** [comment]
work around if anyone else finds this:  
```shell
ansible -b -m shell -a "mkdir /etc/nginx" all 
```

**Comment by traianalinux** [comment]
or better yet:
- name: start nginx-consul service
  service: name=nginx-consul state=started enabled=yes

**Comment by siddharthist** [comment]
This was closed in #1463
